src/VisualStudio/Core/Def/project.json

```diff
@@ -1,86 +1,86 @@
-        "Roslyn.Microsoft.Build": "0.0.2",
-        "NuGet.VisualStudio": {
-            "version": "3.3.0",
-            "suppressParent": "all"
-        }
+  "dependencies": {
+    "ManagedEsent": "1.9.4",
+    "Microsoft.CodeAnalysis.Elfie": "0.10.6-rc2",
+    "Microsoft.VisualStudio.RemoteControl": {
+      "version": "14.0.249-master2E2DC10C",
+      "suppressParent": "all"
     },
+    "Microsoft.VisualStudio.Shell.15.0": {
+      "version": "15.0.25726-Preview5",
+      "suppressParent": "all"
+    },
+    "Microsoft.VisualStudio.Shell.Interop.14.0.DesignTime": {
+      "version": "15.0.25726-Preview5",
+      "suppressParent": "all"
+    },
+    "Microsoft.VisualStudio.Editor": {
+      "version": "15.0.25726-Preview5",
+      "suppressParent": "all"
+    },
+    "Microsoft.VisualStudio.Language.StandardClassification": {
+      "version": "15.0.25726-Preview5",
+      "suppressParent": "all"
+    },
+    "Microsoft.VisualStudio.Language.Intellisense": {
+      "version": "15.0.25726-Preview5",
+      "suppressParent": "all"
+    },
+    "RoslynDependencies.Microsoft.VisualStudio.Text.Internal": {
+      "version": "15.0.25726-Preview5",
+      "suppressParent": "all"
+    },
+    "RoslynDependencies.Microsoft.Internal.VisualStudio.Shell.Interop.14.0.DesignTime": {
+      "version": "14.3.25407",
+      "suppressParent": "all"
+    },
+    "RoslynDependencies.Microsoft.Internal.Performance.CodeMarkers.DesignTime": {
+      "version": "15.0.25726-Preview5",
+      "suppressParent": "all"
+    },
+    "RoslynDependencies.Microsoft.VisualStudio.Progression": {
+      "version": "15.0.25726-Preview5",
+      "suppressParent": "all"
+    },
+    "RoslynDependencies.Microsoft.VisualStudio.GraphModel": {
+      "version": "15.0.25726-Preview5",
+      "suppressParent": "all"
+    },
+    "Microsoft.VisualStudio.ImageCatalog": {
+      "version": "15.0.25726-Preview5",
+      "suppressParent": "all"
+    },
+    "RoslynDependencies.Microsoft.VisualStudio.CallHierarchy.Package.Definitions": {
+      "version": "15.0.25726-Preview5",
+      "suppressParent": "all"
+    },
+    "RoslynDependencies.Microsoft.VisualStudio.Language.CallHierarchy": {
+      "version": "15.0.25726-Preview5",
+      "suppressParent": "all"
+    },
+    "RoslynDependencies.Microsoft.VisualStudio.ComponentModelHost": {
+      "version": "15.0.25726-Preview5",
+      "suppressParent": "all"
+    },
+    "RoslynDependencies.Microsoft.VisualStudio.Diagnostics.PerformanceProvider": {
+      "version": "15.0.25726-Preview5",
+      "suppressParent": "all"
+    },
+    "Microsoft.VisualStudio.Shell.Design": {
+      "version": "15.0.25726-Preview5",
+      "suppressParent": "all"
+    },
+    "RoslynDependencies.Microsoft.VisualStudio.Language.NavigateTo.Interfaces": {
+      "version": "15.0.25726-Preview5",
+      "suppressParent": "all"
+    },
+    "Microsoft.DiaSymReader": "1.1.0-beta1-60625-03",
+    "Roslyn.Microsoft.Build": "0.0.2",
```

Comment:
Ah ok. It is outside the scope of this change, but I wonder if we have enough official packages now to get this working using those instead of one we uploaded (@angocke might know).